test_util/src/tests/envfile.rs

```diff
@@ -0,0 +1,35 @@
+use super::*;
+
+#[test]
+fn create_default() {
+    let expected = format!(
+        "{DEFAULT_TEST_KEY}={DEFAULT_TEST_VALUE}\n{DEFAULT_EXISTING_KEY}={DEFAULT_OVERRIDING_VALUE}",
+    );
+    let actual = create_default_envfile();
+    assert_eq!(
+        expected, actual,
+        "envfile should be created with default values"
```

Comment:
I suggest removing the custom panic message. There is only one point of failure and we can infer what ahppens from the failing test name.